ada-project-docs/wave_03.md

```diff
@@ -1,64 +1,47 @@
-# OPTIONAL Wave 3: Enhancements & Deployment
+# Wave 3: Enhancements & Deployment
 
-These really are **optional** - if you've gotten here and you have time left, that means you're moving speedy fast!
+## Query Parameters
+The following 3 _optional_ query parameters:
```

Comment:
if these are optional should they only go in wave 4?

Reply:
Optional meaning that the user can choose to pass them in or to omit them. Required in that students must implement this functionality. I can add in clarification there.